diff --git a/node_modules/mdast-util-to-markdown/lib/handle/inline-code.js b/node_modules/mdast-util-to-markdown/lib/handle/inline-code.js
index b470685..52346e1 100644
--- a/node_modules/mdast-util-to-markdown/lib/handle/inline-code.js
+++ b/node_modules/mdast-util-to-markdown/lib/handle/inline-code.js
@@ -25,12 +25,14 @@ export function inlineCode(node, _, context) {
 
   // If this is not just spaces or eols (tabs don’t count), and either the
   // first or last character are a space, eol, or tick, then pad with spaces.
-  if (
-    /[^ \r\n]/.test(value) &&
-    (/[ \r\n`]/.test(value.charAt(0)) ||
-      /[ \r\n`]/.test(value.charAt(value.length - 1)))
-  ) {
-    value = ' ' + value + ' '
+  if (/[^ \r\n]/.test(value)) {
+    if (/^[\r\n`]/.test(value)) {
+      value = ' ' + value
+    }
+
+    if (/[\r\n`]$/.test(value)) {
+      value += ' '
+    }
   }
 
   // We have a potential problem: certain characters after eols could result in
